view/frontend/web/template/shipping.html

```diff
@@ -0,0 +1,93 @@
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<li id="shipping" class="checkout-shipping-address" data-bind="fadeVisible: visible()">
+    <div class="step-title" translate="'Shipping Address'" data-role="title"></div>
+    <div id="express-pay-container">
+        <p>Express Checkout</p>
+        <hr/>
+        <div id="express-pay-buttons"></div>
+        <br/>
+        <p>Or</p>
+        <hr/>
+    </div>
+    <div id="checkout-step-shipping"
```

Comment:
This is part of Magento that I'm not super familiar with. Do we need all of this other template code in here? Is there no way to inject the wallet pays without having to include the rest of the wallet pay template? This seems like this could cause theming conflicts

Reply:
Sure lets brainstorm. I was going off the same pattern as what we're doing to inject Fastlane which does this with the e-mail component. Maybe we can find a lower down component as opposed to the shipping.html one to avoid more possible conflicts.

Reply:
Revised the approach and in the latest commit our custom block is injected directly via the xml, no mixin or core templates are being taken over.